contrib/timpi/test/parallel_unit.C

```diff
@@ -123,15 +123,21 @@ std::vector<std::string> pt_number;
     if (TestCommWorld->rank() == 0)
       dest = src;
 
-    TestCommWorld->broadcast(dest);
+    unsigned int root_id = 0;
+#ifdef LIBMESH_HAVE_MPI
+    if (use_proc_null && TestCommWorld->rank() != 0)
+      root_id = (unsigned int)MPI_PROC_NULL;
```

Comment:
These MPI_PROC_NULL uses are going to need to be ifdef-protected too.  Or we could add a TIMPI::null_processor_id shim and use that instead, I suppose?  That would be even better if we'd ever use it, but I can't quickly imagine any algorithms that would need to use MPI_PROC_NULL/null_processor_id in parallel but where we wouldn't just want to ifdef out the entire section of algorithm in serial.